app/authenticated/route.js

```diff
@@ -224,6 +226,14 @@ export default Ember.Route.extend(Subscribe, PromiseToCb, {
     }
   },
 
+  loadRegions() {
+    let svc = this.get('regions');
+    return svc.getAll().then((all) => {
```

Comment:
I don't think you need a service for this, `projects` only has one because it has a bunch of other logic for picking a default and stuff.  `regions: ['userSchemas', this.cbFind('region', userStore')]`